tests/integration/hive_metastore/test_migrate.py

```diff
@@ -663,6 +663,7 @@ def test_migrate_table_in_mount(
     src_external_table = runtime_ctx.make_table(
         schema_name=make_schema(catalog_name="hive_metastore", name=f'mounted_{env_or_skip("TEST_MOUNT_NAME")}').name,
         external_delta=f'dbfs:/mnt/{env_or_skip("TEST_MOUNT_NAME")}/a/b/{tbl_path}',
+        columns=[ColumnInfo(name="0", type_name=ColumnTypeName.STRING)],
```

Comment:
The issue being resolved specifically has a problem with `.` in the column name, so I'd like to see that explicitly covered by the unit test.

In addition I'd like to see a few more examples (similar approach to the tests for `escape_sql_identifier()` routine). Aside from the usual suspects, I'd particularly like to see a column name with <code>&#x60;</code> somewhere in the middle of the name. (That's a corner-case that will fail at the moment, I think.)

Reply:
Alright, thanks, those were indeed not handled. See my changes